### PR TITLE
Math magicans : Setup

### PR DESCRIPTION
**Pull request by o-rashad**
#### Whats new :question:
- Using `Create React App` package a react app has been created.
- MV React linters have been installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (D4nielJ) [inline README.md]: - [x] I think it's better if we use the README file from [Microverse Template](https://github.com/microverseinc/readme-template). Although the one from CRA is not incorrect, it does seem generic and doesn't document the project in a very professional way. I do recommend as well including the scripts from this document in the other.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @od-c0d3r, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- Good use of Create React App
- Everything works as expected :rocket: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (o-rashad) [reply README.md]: working on it :+1:

**Review by elyor-doniyorov — APPROVED** [review]
Hi @od-c0d3r👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**